doc/manual/variables-and-scoping.rst

```diff
@@ -117,38 +134,46 @@ scope through some sort of qualified access.
 The following rules and examples pertain to both hard and soft local
 scopes.  A newly introduced variable in a local scope does not
 back-propagate to its parent scope.  For example, here the ``z`` is not
-introduced into the top-level scope::
+introduced into the top-level scope:
 
-    for i=1:10
-        z = i
-    end
+.. doctest::
+
+    julia> for i=1:10
```

Comment:
I believe I fixed all the spacing issues, including removing the extra indentations in the function declarations at lines 334 and 339. Thanks for all the feedback!